karma.conf.js

```diff
@@ -29,11 +59,7 @@ module.exports = function karma(config) {
     reporters: ['dots'],
 
     files: [
-      'test/*.{jsx,js}',
-    ],
-
-    exclude: [
-      'test/_*.{jsx,js}',
+      'test/*-spec.{jsx,js}',
```

Comment:
Hmm.

tbh I'd rather make a `_helpers` dir, put all the helpers and react-compat under that, and then exclude `_helpers` entirely. I'm not a fan of the "-spec" convention, and if we want it enshrined in config, we'd need to have a way to have it break if someone forgets to add the suffix.